qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -13,9 +13,9 @@
 class CalibrationModel(object):
     # model shares code with the EnginX auto reduction script - code is kept in EnggUtils.py
     @staticmethod
-    def create_new_calibration(calibration, rb_num, plot_output, save_dir=output_settings.get_output_path()):
+    def create_new_calibration(calibration, rb_num, plot_output, save_dir=output_settings.get_output_path):
         full_calib = load_full_instrument_calibration()
-        EnggUtils.create_new_calibration(calibration, rb_num, plot_output, save_dir, full_calib)
+        EnggUtils.create_new_calibration(calibration, rb_num, plot_output, save_dir(), full_calib)
```

Comment:
Have you tried to run this function with `savedir='/tmp/'` argument? On my toy model I get `TypeError: 'str' object is not callable`